controllers/backplaneconfig_controller_test.go

```diff
@@ -52,7 +52,7 @@ var _ = Describe("BackplaneConfig controller", func() {
 	clusterManager.SetGroupVersionKind(schema.GroupVersionKind{Group: "operator.open-cluster-management.io", Version: "v1", Kind: "ClusterManager"})
 	hiveConfig := &unstructured.Unstructured{}
 	hiveConfig.SetGroupVersionKind(schema.GroupVersionKind{Group: "hive.openshift.io", Version: "v1", Kind: "HiveConfig"})
-
+	os.Setenv("POD_NAMESPACE", "default")
```

Comment:
Id recommend moving this line to this function -
https://github.com/open-cluster-management/backplane-operator/blob/main/controllers/suite_test.go#L83
to ensure it runs before the suite.

It can also be unset after -
https://github.com/open-cluster-management/backplane-operator/blob/main/controllers/suite_test.go#L111